Makefile

```diff
@@ -0,0 +1,48 @@
+.PHONY: help
+
+#
+# Configuration variables
+#
+VENVDIR?=.venv
+VENV = $(VENVDIR)/bin
+
+PYTHON3OK :=$(shell python3 --version 2>&1)
+ifeq ($(PYTHON3OK),)
+  brew -y install python3.6 python3-pip
```

Comment:
I don't think `brew` should be assumed and technically Makefiles should run on Linux, i.e. on Travis/CI. 

I would not install Python in the Makefile. The listed project prereqs, like Python, `kubectl`, `tkn` CLI should be installed by the user since he/she need to make conscious choices about those.

Setting up the virtualenv is kind of a special case here in order to encapsulate installing required packages and not mess up any of the user's environment.